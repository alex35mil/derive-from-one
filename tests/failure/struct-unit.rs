use derive_from_one::FromOne;

#[derive(FromOne)]
pub struct T;

fn main() {}
