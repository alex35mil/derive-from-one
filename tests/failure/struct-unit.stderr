error: FromOne can only be used with structs with a single field
 --> tests/failure/struct-unit.rs:4:12
  |
4 | pub struct T;
  |            ^
